specification/security/resource-manager/Microsoft.Security/preview/2020-01-01-preview/securityContacts.json

```diff
@@ -67,9 +67,6 @@
               "$ref": "../../../common/v1/types.json#/definitions/CloudError"
             }
           }
-        },
```

Comment:
the service doesn't support pagination at all, nor it makes sense to change it to support it now - this change is required.